src/qibocal/protocols/characterization/flux_dependence/utils.py

```diff
@@ -172,6 +184,54 @@ def flux_dependence_plot(data, fit, qubit):
     return figures, fitting_report
 
 
+def flux_crosstalk_plot(data, fit, qubit):
+    figures = []
+    fitting_report = ""
+
+    all_qubit_data = data[qubit]
+
+    fig = make_subplots(
+        rows=1,
+        cols=len(all_qubit_data),
+        vertical_spacing=0.1,
+        subplot_titles=len(all_qubit_data) * ("MSR [V]",),
+    )
+
+    for col, (flux_qubit, qubit_data) in enumerate(all_qubit_data.items()):
+        frequencies = qubit_data.freq * HZ_TO_GHZ
+        msr = qubit_data.msr
+        if data.resonator_type == "2D":
+            msr = -msr
+
+        fig.add_trace(
+            go.Heatmap(
+                x=frequencies,
+                y=qubit_data.bias,
+                z=qubit_data.msr * V_TO_UV,
+            ),
+            row=1,
+            col=col + 1,
+        )
+
+        fig.update_xaxes(
+            title_text=f"{qubit}: Frequency (Hz)",
```

Comment:
```suggestion
            title_text="Frequency (Hz)",
```
I think that the label is clear also without `qubit`